pruebas.py

```diff
@@ -0,0 +1,21 @@
+from os import fwalk
+from src.utils.validation import *
+from src.models.coordenada import *
+from src.models.fecha import *
+from src.models.estado import *
+from src.models.fwi import *
+from src.enums.day import *
+from src.enums.month import *
+
+
+def testeo():
```

Comment:
Esto no lo podéis hacer. Primero porque no sigue las buenas prácticas, segundo, porque no comprueba la sintaxis.